content/docs/lists-and-keys.md

```diff
@@ -296,6 +296,6 @@ function NumberList(props) {
 }
 ```
 
-[**Pruebalo en CodePen**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/BLvYrB?editors=0010)
 
-Algunas veces esto resulta en codigo mas claro, pero este estilo puede ser tambien abusado. Como en JavaScript, depende de ti decidir cuando vale la pena extraer una variable por legibilidad. Ten en mente que si el cuerpo de `map()` esta muy anidado, puede ser un buen momento para [extraer un componente](/docs/components-and-props.html#extracting-components).
+Algunas veces esto resulta en código mas claro, pero este estilo puede ser también abusado. Como en JavaScript, depende de ti decidir cuando vale la pena extraer una variable por legibilidad. Ten en mente que si el cuerpo de `map()` esta muy anidado, puede ser un buen momento para [extraer un componente](/docs/components-and-props.html#extracting-components).
```

Comment:
I think `también puede ser  abusado` is better then `puede ser también abusado`.

Reply:
What about `puede ser abusado tambien?`

Reply:
I think `abusado` ought to be first, then the rest of the phrase should follow.